docs/pages/inboxes/device-sync.md

```diff
@@ -0,0 +1,79 @@
+# Enable device sync for apps built with XMTP
+
+Enable the device sync feature to give your users a way to sync decrypted historical data from an existing app installation on one device—with a new app installation on another device.
+
+This historical data includes:
+
+- Conversations
+- Conversation messages
+- Consent state
+- HMAC keys (for push notifications)
+
+Device sync enables your users pick up conversations where they left off, regardless of the app installation they use. All they need is an existing installation online to provide the data.
+
+## Device sync components
+
+Device syncs are accomplished using these components:
+
+- Sync group
+- Sync worker
+- History server
+
+### Sync group
+
+A sync group is a special [Messaging Layer Security](/protocol/specs) group that includes all of the user’s devices. The sync group is used to send serialized sync messages across the user's devices. The sync group is filtered out of most queries by default so they don’t appear in the user’s inbox.
+
+### Sync worker
+
+A sync worker is a spawned background worker that listens for sync events, and processes them accordingly. This worker:
+
+- Emits and updates consent
+- Creates and consumes archive payloads for and from other devices
+- Keeps preferences synced across devices
+
+### History server
+
+A history server acts as a bucket that holds encrypted sync payloads. The URL location of these payloads and the password (cipher encryption key) to decrypt these payloads is sent over the sync group for the recipient to decrypt.
+
+## How device sync works
+
+When your app initializes an XMTP client and a `historySyncUrl` client option is present, device sync automatically triggers an initial sync request, pulling all message history into the new app installation and merging it with the existing app installations in the sync group.
```

Comment:
Is "message history" OK here? Do we want to say "historical data" instead so that it includes all of the data types listed above? Or is it really just message history?